packages/flutter/lib/src/material/reorderable_list.dart

```diff
@@ -66,14 +103,14 @@ class ReorderableListView extends StatefulWidget {
     this.scrollDirection = Axis.vertical,
     this.padding,
     this.reverse = false,
-  }) : assert(scrollDirection != null),
-       assert(onReorder != null),
-       assert(children != null),
-       assert(
-         children.every((Widget w) => w.key != null),
-         'All children of this widget must have a key.',
-       ),
-       super(key: key);
+  })  : assert(scrollDirection != null),
```

Comment:
@sanjoli63 Please discard these changes and save a file without formatting.
Thanks.